README.rst

```diff
@@ -112,7 +112,7 @@ You can run **conan** client and server in Windows, MacOS, and Linux.
       search     Searches package recipes and binaries in the local cache or in a remote.
     Creator commands
       new        Creates a new package recipe template with a 'conanfile.py'.
-      create     Builds a binary package for recipe (conanfile.py) located in current dir.
+      create     Builds a binary package for a recipe (conanfile.py) located in the current dir.
```

Comment:
These might be automatically obtained from the ``conan --help`` command, maybe it would make sense to fix them in the app code.